src/Tribe/Views/V2/View.php

```diff
@@ -1146,7 +1148,20 @@ protected function setup_repository_args( Context $context = null ) {
 
 		// Set's up category URL for all views.
 		if ( ! empty( $context_arr[ TEC::TAXONOMY ] ) ) {
-			$args[ TEC::TAXONOMY ] = $context_arr[ TEC::TAXONOMY ];
+			if ( ! is_array( $context_arr[ TEC::TAXONOMY ] ) ) {
+				$args[ TEC::TAXONOMY ] = $context_arr[ TEC::TAXONOMY ];
+			} else {
+				if ( empty( $args['tax_query'] ) ) {
+					$args['tax_query'] = [];
+				}
+
+				$args['tax_query'][] = [
+					'taxonomy' => TEC::TAXONOMY,
+					'field'    => 'slug',
+					'terms'    => $context_arr[ TEC::TAXONOMY ]
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: ec0cab995acc1dee11dba2cd150943dd</sub>